### PR TITLE
added any to paddle math

### LINKED ISSUE #19552: any

### PR DESCRIPTION
**Pull request by sroycho4**
close #19552

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/math.py [hunk 1]
```diff
@@ -384,3 +384,8 @@ def lerp(x, y, weight, name=None):
 @to_ivy_arrays_and_back
 def rsqrt(x, name=None):
     return 1 / ivy.sqrt(x)
+
+
```
Comment (rishabgit) [inline ivy/functional/frontends/paddle/tensor/math.py:388]: missing supported dtypes

File: ivy/functional/frontends/paddle/tensor/math.py [hunk 2]
```diff
@@ -384,3 +384,8 @@ def lerp(x, y, weight, name=None):
...
+@to_ivy_arrays_and_back
+def any(x, axis=None, keepdim=False, name=None):
+    return ivy.max(x, axis=axis, keepdims=keepdim)
```
Comment (rishabgit) [inline ivy/functional/frontends/paddle/tensor/math.py:391]: not sure why it's calling ivy.max instead of ivy.any?

**Review by rishabgit — CHANGES_REQUESTED** [review]
Hey @sroycho4, thanks for looking into this! :smile:
Added comments in the code, feel free to request a review after making the changes. 
Hope this helps - https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/any_en.html
Thanks! :smile:

**Comment by sroycho4** [comment]
@rishabgit whoops, don't know how it got altered. Fixed them!

**Review by rishabgit — CHANGES_REQUESTED** [review]
Still missing supported dtypes :sweat_smile: 
Feel free to request a re-review!

**Comment by sroycho4** [comment]
@rishabgit done!

**Comment by rishabgit** [comment]
Thanks for the changes, they look great!

**Pull request merged**